plugins-list.yaml

```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
+# plugins/events-backend-module-github ==> Added as a static plugin in RHDH
+# plugins/events-backend-module-gitlab ==> Added as a static plugin in RHDH
+plugins/events-backend-test-utils:
+plugins/events-node:
 plugins/example-todo-list:
 plugins/example-todo-list-backend:
+plugins/example-todo-list-common:
 plugins/home:
-plugins/kubernetes:
-plugins/kubernetes-backend:
+plugins/home-react:
+# plugins/kubernetes  ==> wrapper
+# plugins/kubernetes-backend  ==> wrapper
+plugins/kubernetes-cluster:
+plugins/kubernetes-common:
+plugins/kubernetes-node:
+plugins/kubernetes-react:
 plugins/notifications:
-plugins/notifications-backend: --embed-package @backstage/plugin-signals-node
+plugins/notifications-backend:
+plugins/notifications-backend-module-email:
+plugins/notifications-common:
+plugins/notifications-node:
 plugins/org:
-#plugins/permission-backend: ==> Added as a static plugin in RHDH
-#plugins/permission-backend-module-policy-allow-all: ==> Not meaningful for RHDH
+plugins/org-react:
+# plugins/permission-backend ==> Added as a static plugin in RHDH
+# plugins/permission-backend-module-policy-allow-all ==> Added as a static plugin in RHDH
+plugins/permission-common:
+plugins/permission-node:
+plugins/permission-react:
 plugins/proxy-backend:
-#plugins/scaffolder: ==> Added as a static plugin in RHDH
-#plugins/scaffolder-backend: ==> Added as a static plugin in RHDH
-plugins/scaffolder-backend-module-azure:
+# plugins/scaffolder  ==> Added as a static plugin in RHDH
+# plugins/scaffolder-backend  ==> Added as a static plugin in RHDH
+# plugins/scaffolder-backend-module-azure  ==> wrapper
 plugins/scaffolder-backend-module-bitbucket:
-plugins/scaffolder-backend-module-bitbucket-cloud:
-plugins/scaffolder-backend-module-bitbucket-server:
+# plugins/scaffolder-backend-module-bitbucket-cloud  ==> wrapper
+# plugins/scaffolder-backend-module-bitbucket-server  ==> wrapper
 plugins/scaffolder-backend-module-confluence-to-markdown:
 plugins/scaffolder-backend-module-cookiecutter:
-plugins/scaffolder-backend-module-gerrit:
+plugins/scaffolder-backend-module-gcp:
+# plugins/scaffolder-backend-module-gerrit  ==> wrapper
 plugins/scaffolder-backend-module-gitea:
-plugins/scaffolder-backend-module-github:
-plugins/scaffolder-backend-module-gitlab:
+# plugins/scaffolder-backend-module-github  ==> wrapper
+# plugins/scaffolder-backend-module-gitlab  ==> wrapper
+plugins/scaffolder-backend-module-notifications:
 plugins/scaffolder-backend-module-rails:
 plugins/scaffolder-backend-module-sentry:
 plugins/scaffolder-backend-module-yeoman:
-#plugins/search: ==> Added as a static plugin in RHDH
-#plugins/search-backend: ==> Added as a static plugin in RHDH
+plugins/scaffolder-common:
+plugins/scaffolder-node:
+plugins/scaffolder-node-test-utils:
+plugins/scaffolder-react:
+# plugins/search  ==> Added as a static plugin in RHDH
+# plugins/search-backend  ==> Added as a static plugin in RHDH
 plugins/search-backend-module-catalog:
```

Comment:
Some of these search modules are probably included in the RHDH already aren't they ?

Reply:
I could not find it in wrappers, where else can I check?

Reply:
in backend/index.ts, where static plugins are wired